### DISCUSSION TITLE
Differences on output between IDE and command line for 'coverage' operation.

### DISCUSSION THREAD

**Discussion by DKnoto** [albilu/netbeansPython#33] [General]
For a trivial project with the following directory structure:

`main.py`
 `tests/FuncTest.py`

I ran the operation `Project Name/Code Coverage/Collect and Display Code Coverage`. On the `Code Coverage` bar, I launched the `All Tests` operation. A message appeared in the `Output/Collect Coverage` window: `Wrote JSON report to coverage.json`. I then ran the `Report...` operation and got a message in the report window that the code has zero test coverage: `No data -- have you run your code yet? | 0.00%`

In the terminal window in the project directory, I verified that `.coverage` and `coverage.json` files were created. I ran the `coverage report -m` command from the command line and got the following results:
```
Name     Stmts  Miss  Branch  BrPart  Cover   Missing
-----------------------------------------------------
main.py     20    15      10       0    17%   19-39
-----------------------------------------------------
TOTAL       20    15      10       0    17%
```

**Comment by albilu** [comment]
Thanks you for reporting this. A small regression was introduced after a recent refactoring. 
This is now fixed and will be available in next release.
